app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    old_file = command.split()[1]
+    new_file = command.split()[2]
```

Comment:
Don't use split twice. You can create a separate variable and store a result of `command.split` in this variable and use it where you need.